ui/components/MesheryPatterns.js

```diff
@@ -658,11 +658,12 @@ function MesheryPatterns({
     }
   });
 
-  async function showModal(count) {
+  async function showModal(count, patterns) {
+    console.log("patterns to be deleted", count, patterns);
     let response = await modalRef.current.show({
       title : `Delete ${count ? count : ""} Pattern${count > 1 ? "s" : ''}?`,
 
-      subtitle : `Are you sure you want to delete ${count > 1 ? "these" : 'this'}  ${count ? count : ""}  pattern${count > 1 ? "s" : ''}?`,
+      subtitle : `Are you sure you want to delete ${count > 1 ? "these" : 'this'} ${patterns} pattern${count > 1 ? "s" : ''}?`,
```

Comment:
```suggestion
      subtitle : `Are you sure you want to delete ${count > 1 ? "these" : 'this'} ${patterns} pattern${count > 1 ? "s" : ''}?`,
```
There's no need for "this" and "these", you can change it to "the  (pattern-name) design or designs"

Reply:
final updated ss 

![image](https://user-images.githubusercontent.com/95926324/181471214-1ba2a9c6-c987-4026-ae2c-6854e045f281.png)